queries.sql

```diff
@@ -1,3 +1,16 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT * FROM animals WHERE neutered IS true AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered IS true;
+
+SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] As it is a strict requirement that you include screenshots of your query results in your pull request, kindly ensure that you include them.

![image](https://user-images.githubusercontent.com/86318284/215475355-719c9612-1267-4126-b572-f4c8e891be0d.png)

Reply:
Hi, 
okay, I will add it.
 thanks